pandas/core/generic.py

```diff
@@ -2458,8 +2462,17 @@ def to_msgpack(self, path_or_buf=None, encoding='utf-8', **kwargs):
         return packers.to_msgpack(path_or_buf, self, encoding=encoding,
                                   **kwargs)
 
-    def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
-               index_label=None, chunksize=None, dtype=None, method=None):
+    # TODO: Replace `Callable[[Any, Any, ...` when SQLTable and sqlalchemy
+    # can be imported. SQLTable can't be imported due to circular import.
+    # sqlalchemy can't be imported since it's an optional dependency.
+    def to_sql(self, name: str, con,
```

Comment:
Sorry should have asked this before but can you put each parameter on a separate line? Will help with readability

Reply:
Done. Also added `Any` to `con=` and added explanation to the note so that less thinking is required later.